targets/corefx.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <ProjectDirectory Condition="'$(PathToRepo)' != ''">$(PathToRepo)</ProjectDirectory>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration) -buildArch=$(Platform) -portable=false -- /p:ILLinkTrimAssembly=false /p:BuildAllPackages=true</BuildCommand>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration) -buildArch=$(Platform) -portable=false -allConfigurations -- /p:ILLinkTrimAssembly=false</BuildCommand>
```

Comment:
To me, it seems like corefx should be checking the `$(DotNetBuildFromSource)` flag here, instead of source-build having to specify `-allConfigurations`.

Do you plan on doing this later?

Reply:
I agree and plan to do it later, when we start to solidify the exact command lines we pass to the repos.

Reply:
Since this is a servicing branch, don't you want `/p:BuildAllPackages=true`?  Or does `-allConfigurations` do the same thing?